src/index.jsx

```diff
@@ -0,0 +1,6 @@
+import React from "react";
```

Comment:
Does changing this to a .jsx file affect how we will be exporting the Astro module in the future?

Reply:
Possibly, but I can always change it to JS and it should still compile.  Kinda waiting to see how it will all come together once we have all the code together.